app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -583,17 +598,9 @@ open class DefaultToolbarMenu(
 
     private fun handleBookmarkItemTapped() {
         if (!isCurrentUrlBookmarked) isCurrentUrlBookmarked = true
-        onItemTapped.invoke(ToolbarMenu.Item.Bookmark)
+        onItemTapped.invoke(Item.Bookmark)
     }
 
-    @ColorRes
```

Comment:
Why did we move this code from here to above in this bug fix? How does it relate to this sign-in bug?

Reply:
They are moved to a different place. Code cleanup.

Reply:
Let's avoid code cleanup and refactors in bug fixes. See https://github.com/mozilla-mobile/fenix/pull/19118#discussion_r616128770.